src/API/QueryFieldService.php

```diff
@@ -73,6 +74,31 @@ public function countContentItems(Content $content, string $fieldDefinitionIdent
         return $this->searchService->findContent($query)->totalCount;
     }
 
+    public function loadContentItemsSlice(Content $content, string $fieldDefinitionIdentifier, int $offset, int $limit): iterable
+    {
+        $query = $this->prepareQuery($content, $fieldDefinitionIdentifier);
```

Comment:
Could we make `offset` and `limit` available during the parameters resolving? (https://github.com/ezsystems/ezplatform-query-fieldtype/blob/566bf60ca149757b8dddb908dd61c5a44077c1a8/src/API/QueryFieldService.php#L145-L153)

Use case: QueryType which call 3rd party system (e.g. recommendation service) to get content id's and then return s the following query:  

```php
$query = new Query();
$query->filter = new ContentId($idsFrom3rdPartySystem);
```

Reply:
You mean as variables that can be used by the expression language ? I need to look into it, maybe. Interesting idea.

Reply:
Done, `@=offset` and `@=limit` are now available for resolving *when pagination is enabled*.